de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/PurchaseOrderToShipperTransportationRepository.java

```diff
@@ -0,0 +1,98 @@
+package de.metas.order;
+
+import static org.adempiere.model.InterfaceWrapperHelper.save;
+
+import java.sql.Timestamp;
+
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.ad.dao.IQueryFilter;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_M_Package;
+import org.springframework.stereotype.Repository;
+
+import com.google.common.collect.ImmutableList;
+
+import de.metas.adempiere.model.I_C_Order;
+import de.metas.shipping.api.IShipperTransportationBL;
+import de.metas.shipping.api.IShipperTransportationDAO;
+import de.metas.shipping.model.I_M_ShipperTransportation;
+import de.metas.shipping.model.I_M_ShippingPackage;
+import de.metas.shipping.model.ShipperTransportationId;
+import de.metas.util.Services;
+import lombok.NonNull;
+
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+@Repository
+public class PurchaseOrderToShipperTransportationRepository
```

Comment:
looking at the code this is not a repository.
in my mind a repository is something where you can load and save entities.
@teosarca wdyt, is it sufficient to simply rename this to something that ends with `...Service`?

Reply:
..or mayby `..DAO`, sinse we already have a service which is more BL'ish, compared to this class

Reply:
...oooor maybe we can start with a small shipperTeansportation entity...
we could have a repo for it..and it could have a purchase order ids, and code such as `C_Order_AddTo_M_ShipperTransportation` could add another one, instead of having this `addPurchaseOrderToShipperTransportation`

And then the repo could save it, by behind the scenes creating that `I_M_Package` and that `I_M_ShippingPackage`